java/src/Ice/src/main/java/com/zeroc/Ice/InputStream.java

```diff
@@ -962,17 +964,17 @@ public byte readByte()
      * Extracts an optional byte value from the stream.
      *
      * @param tag The numeric tag associated with the value.
-     * @return The optional value (if any).
+     * @return The extracted byte value, or null if it wasn't sent.
      **/
-    public java.util.Optional<Byte> readByte(int tag)
+    public @Nullable Byte readByte(int tag)
     {
         if(readOptional(tag, OptionalFormat.F1))
```

Comment:
Should we switch to the more compact ` return readOptional(...) ? readByte() : null;` ?